metaci/release/utils.py

```diff
@@ -96,8 +97,8 @@ def send_release_webhook(project_config, release, config_item=None):
 
     payload = {
         "case_template_id": release.change_case_template.case_template_id,
-        "package_name": project_config.project__package__name,
-        "version": project_config.get_version_for_tag(tag),
+        "package_name": release.repo.name,  # Need to figure out.
```

Comment:
This is probably okay. In the future we might want to add a field on Repository so that we can configure a project name that is different from the repo name.